EIPS/eip-3668.md

```diff
@@ -3,7 +3,7 @@ eip: 3668
 title: "CCIP Read: Secure offchain data retrieval"
```

Comment:
While not really related to this PR, what does CCIP stands for? 

My guess is "Client-Contract Interaction Protocol" or something similar. I think it would be nice to resolve it in the abstract or description.

Reply:
Cross Chain Interoperability Protocol - https://chain.link/cross-chain